src/main/java/impl/weather/tempInfoComp.java

```diff
@@ -0,0 +1,12 @@
+package impl.weather;
+
+import api.weather.DayTemperatureInfo;
+
+import java.util.Comparator;
+
+public class tempInfoComp implements Comparator<DayTemperatureInfo> {
```

Comment:
codestyle

Reply:
What do you mean ?? :(

Reply:
Read code conventions, please, https://www.oracle.com/technetwork/java/codeconvtoc-136057.html

Reply:
thanks but I still don't know what is the problem with these lines ?? and others? could you please just specify the problem

Reply:
???

Reply:
Oh, excuse me, I missed it.  https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367